src/main/java/me/zeroeightsix/kami/module/modules/combat/VisualRange.kt

```diff
@@ -40,7 +41,7 @@ class VisualRange : Module() {
 
         if (tickPlayerList.size > 0) {
             for (playerName in tickPlayerList) {
-                if (playerName == mc.player.name) continue
+                if ((playerName == mc.player.name) || (Friends.isFriend(playerName) && !friends.value)) continue
```

Comment:
oh sorry for the trouble, but could you change it to 
```kotlin
if ((playerName == mc.player.name) || (!friends.value && Friends.isFriend(playerName))) continue
```
i just realised, it would be better on performance if you had the boolean first